server/tracedb/tracedb_test.go

```diff
@@ -68,7 +68,7 @@ func TestCreateClient(t *testing.T) {
 			cl := c
 			t.Parallel()
 
-			actual, err := tracedb.New(cl.config)
+			actual, err := tracedb.New(cl.config, nil)
```

Comment:
let's fix this `New` method (in the future) so it doesn't require all the deps regardless of the final type being created